### ISSUE TITLE
Get-GitHubRepoLogs and Get-GitHubVulnerabilityAlerts playbooks only process the first 30 repositories in an organisation

### ISSUE THREAD

**Issue by kirk-spragg** [Azure/Azure-Sentinel#1087]
**Describe the bug**
I have deployed and configured the two github monitoring playbooks as described  [here](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/GitHub)

This all works, have also deployed the parsers as described in the same link above. So far so good.

Now upon inspecting the results in my Azure Sentinel workspace, it is clear that only 30 of our organization's ~430 or so repositories have been monitored. This appears to be due to an oversight in bot Get-GitHubRepoLogs and Get-GitHubVulnerabilityAlerts. These logic apps (deployed as part of the instructions I linked to at the top) only fetch the first page of repositories from https://api.github.com/orgs/@{variables('OrgName')}/repos and stops, there appears to be no logic to fetch and parse more than the first page of results from the github api.

**To Reproduce**
1 - Have a github org with more than 30 repositories
2 - Follow the instructions in this repo ([here](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/GitHub))
3 - Only 30 repositories are actually monitored by Get-GitHubRepoLogs & Get-GitHubVulnerabilityAlerts and logging in the azure sentinel workspace only contains results for these 30 repositories

**Expected behavior**
Results form all of the repositories in the github org will be monitored and log entries for them will appear in the sentinel workspace

**Screenshots**
![image](https://user-images.githubusercontent.com/26468285/93282280-274aa000-f822-11ea-8376-ed1bc00bd795.png)

**Comment by dicolanl** [comment]
@kirk-spragg  wow 400+ ORGS! 
Any chance you can share the output of that step so i can see what the body looks like with 30+ orgs?

we can do it privately if needed.

**Comment by kirk-spragg** [comment]
@dicolanl No we have one org with that's 400+ repositories!

The step in the screenshot is the step that performs the github api call to get the repos from our org (we only have one).

However that endpoint (https://api.github.com/orgs/seequent/repos in our case) only returns the first 30 repo in our org, the other 420 or so are on subsequent pages, but these pages are not fetched by the logic app.

**Comment by dicolanl** [comment]
@kirk-spragg  yup sorry meant repos.  I understand what the HTTP call is doing.  but i need to see the response when there is more than 30 so I can add logic to handle the next 30.  Can you share here? or in PM?

**Comment by kirk-spragg** [comment]
@dicolanl I'll add a number of empty public repos to our github org (https://github.com/seequent), we already have 28 so if I add 5 or 6 more that will push it ove the page limit and you'll be able to work directly with the github api from our organisation.

Not willing to give you an api response containing references to a number of private repos sorry, but this should be enough.

Will let you know when I've added the additional public repos.

**Comment by dicolanl** [comment]
cool makes sense.

**Comment by kirk-spragg** [comment]
@dicolanl I've added a number of empty public repositories there are now 33 on our org so you should be good to go.

**Comment by dicolanl** [comment]
@kirk-spragg  we are still working on this.  we updated the logic to take more than 30 repos, but found another bug we are working on.

**Comment by kirk-spragg** [comment]
@dicolanl Thanks for the update, please let me know when this is resolved.

**Comment by dicolanl** [comment]
#1121  resolves this

**Comment by preetikr** [comment]
@kirk-spragg - Please confirm if this resolves your issue. Thanks.

**Comment by kirk-spragg** [comment]
@preetikr I'll take a look today (its friday here) or monday depending on my vailability.

**Comment by kirk-spragg** [comment]
@preetikr  What is the simplist way to apply these changes to an existing azure deployment of the playbooks?

**Comment by kirk-spragg** [comment]
@preetikr  @dicolanl There appears to a bug with Get-GitHubVulnerabilityAlerts. The final Untill loop only runs once and only runs on the LAST repository processed in the fore each loop above it.

![image](https://user-images.githubusercontent.com/26468285/94864657-d3ce8800-0498-11eb-8cd4-c59c065057f4.png)

![image](https://user-images.githubusercontent.com/26468285/94864727-e8ab1b80-0498-11eb-87d8-a73ce47bda20.png)

**Comment by dicolanl** [comment]
@kirk-spragg 
I believe you can just redeploy the arm template and it should update the logic app.

yup.  simple fix.  DRAG the until loop upwards into the for each loop.  we will update the code, but you dont need to wait for that
![image](https://user-images.githubusercontent.com/25964057/94866873-fad37e00-03f4-11eb-9642-08d33ca6b56c.png)

![image](https://user-images.githubusercontent.com/25964057/94866884-fe670500-03f4-11eb-95f5-0707ac25bfe0.png)

**Comment by kirk-spragg** [comment]
@dicolanl While looking at run results shows the "Get-GitHubVulnerabilityAlerts" logic app in graphical form, attemting to edit it in the logic apps designer in azure portal fails with  "Operation failed: a is undefined" and I can only see the code view.

This isn't an issue with the other two logic apps. I re-deployed the entire environment by following the steps here https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/GitHub
so basically a new/clean deployment from scratch.

![image](https://user-images.githubusercontent.com/26468285/94871288-25c9da80-04a6-11eb-9fcf-0df79f7ec370.png)

**Comment by sreedharande** [comment]
@kirk-spragg - This issue has been fixed on Oct 4th with PR# 1145 https://github.com/Azure/Azure-Sentinel/pull/1145. Did you get a chance to review this.

**Comment by sarah-yo** [comment]
Fixed with PR #1145
